src/Run/Setup.cs

```diff
@@ -4,20 +4,56 @@
 
 using System;
 using System.IO;
+using System.Linq;
 using System.Collections.Generic;
 using System.Text;
 
 namespace Microsoft.DotNet.Execute
 {
     public class Setup
     {
+        private const string RunQuietReservedKeyword = "RunQuiet";
+        private const string RunToolSettingValueTypeReservedKeyword = "runToolSetting";
+
+        public Dictionary<string, string> ToolSettings { get; set; }
         public Dictionary<string, Setting> Settings { get; set; }
         public Dictionary<string, Command> Commands { get; set; }
         public Dictionary<string, Tool> Tools { get; set; }
         public Dictionary<string, string> SettingParameters { get; set; }
         public string Os { get; set; }
         public string ConfigurationFilePath { get; set; }
 
+        private int ValidateSettings()
+        {
+            int returnCode = 0;
```

Comment:
I figured this would get fleshed out more in the future and that there would be a possibility for having other returnCodes for warnings vs errors or something else.  Given that doesn't represent the current state of the code, I'm ok with making this a bool if it's confusing.